LPS-125709 Staging publish page display in half page

moltam89 opened the pull request.

rodrigopaulino committed:
LPS-122906 Make sure to append the latest version of the workflow definition when updating the link

rodrigopaulino committed:
LPS-122906 Adapting tests to new behavior

robertoDiaz committed:
LPS-125733 Allow pass the sidenavId to the taglib

robertoDiaz committed:
LPS-125733 Avoid repeat ids in the panel

pietrobocale committed:
COMMERCE-5486 Add a pause to the macro addNewChannelsEntry

georgel-pop-lr committed:
LPS-124597 [7.4 only] Image resize function not working in ItemSelector

This commit fix resizing functionality broken in commit
fb3d12c8032e242708661b59b41d035847c08c75. The cause is the use of
`insertElement()` that seems to not add the selection outer element on
the image.

The original fix proposed here:

     https://github.com/liferay-frontend/liferay-portal/pull/720

was careful to preserve the IE-specific behavior, but
given that we no longer need to support IE in 7.4 we take the
opportunity to simplify the code instead.

Commit:
LPS-125256 Remove deprecated tag attributes

Deprecated attributes have been removed from the following tags and
there usages:

- `<clay:alert />`
- `<clay:badge />`
- `<clay:button />`
- `<clay:checkbox />`
- `<clay:dropdown-actions />`
- `<clay:dropdown-menu />`
- `<clay:icon />`
- `<clay:label />`
- `<clay:link />`
- `<clay:navigation-bar />`
- `<clay:progressbar />`
- `<clay:radio />`
- `<clay:sticker />`
- `<clay:stripe />`

Commit:
LPS-126602 To update ResetImportedStructureDefaultValues in order to test cancelation. Pending POSHI-12 resolution

Commit:
LPS-126657 To update ResetStructureDefaultValues in order to test  cancelation. Pending POSHI-12 resolution

Commit:
LPS-126657 SF

Commit:
LPS-126657 Delete stub and update test messages

joseabelenda committed:
LPS-126032 - Checking if isStagingInProcess()

kenjiheigel committed:
POSHI-146 Update default poshi runner to 1.0.296

kenjiheigel committed:
POSHI-146 Generate changelog

jorgediaz-lr committed:
LPS-127109 New DefaultUser tests for segments that contains both context and model criteria with 'and' and 'or' conjunctions. Same tests are also added for non matching and without adding the SIGNED_IN parameter in the context.

jorgediaz-lr committed:
LPS-127109 If we are the guest user and the model criterion is not empty, isMember always has to return false

riccardo-alberti committed:
COMMERCE-5579 Add cpDefinitions portlet to resource actions

riccardo-alberti committed:
COMMERCE-5579 Use correct method in service to check permissions

michaelhashimoto committed:
LRCI-1931 Ignore test classes when they will not be ran

michaelhashimoto committed:
LRCI-1931 Use the java file instead of the class file for junit test class groups

michaelhashimoto committed:
LRCI-1931 Update how to identify poshi test results

michaelhashimoto committed:
LRCI-1931 Change how to get the test class results

juliaduartelira committed:
LPS-126837 Refactor ExcludeRepeatedRichTextOnWebContent, RepeatRichTextOnWebContent

juliaduartelira committed:
LPS-126837 Add RepeatRichTextOnUpload

pedro-michel committed:
LPS-126848 Add VerifyRichTextIsSetAsRequiredOnUploading

pedro-michel committed:
LPS-126848 SF

michaelhashimoto committed:
LRCI-1931 SF

michaelhashimoto committed:
LRCI-1931 Change how to get test results from builds

michaelhashimoto committed:
LRCI-1931 Change how to get cucumber test results from builds

Commit:
LPS-125709 Partially revert LPS-125446

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

moltam89 commented:
ci:test:sf

liferay-continuous-integration commented:
Your pull request was not tested because your repository has not set liferay-continuous-integration as a collaborator. See https://grow.liferay.com/share/Pull+Request+Tester+for+Liferay+Developers for more information.

moltam89 commented:
ci:test:relevant

moltam89 commented:
ci:test:stable

liferay-continuous-integration commented:
Your pull request was not tested because your repository has not set liferay-continuous-integration as a collaborator. See https://grow.liferay.com/share/Pull+Request+Tester+for+Liferay+Developers for more information.

liferay-continuous-integration commented:
Your pull request was not tested because your repository has not set liferay-continuous-integration as a collaborator. See https://grow.liferay.com/share/Pull+Request+Tester+for+Liferay+Developers for more information.

marcosapmf committed:
LPS-126471 normalize option value for all available languages

Commit:
LPS-126829 Add EmptyAFieldGroup

Commit:
LPS-126829 Add DataEngineBuilder

Commit:
LPS-126829 SF

naraandrade committed:
LPS-127138 Adjust lang keys

naraandrade committed:
LPS-127138 buildLang

Commit:
LPS-126390 Add TranslationDoesNotChangeAnother

Commit:
LPS-126390 SF

BryanEngler committed:
LPS-127476 portal-search-elasticsearch7-impl: Process IndexSettingsContributors on activation and guarantee that default mapping is always last during template merge

https://issues.liferay.com/browse/LPS-127476

Commit:
LPS-126375 Add test DefaultTitlePersistsUponFirstlyTranslatingForAnotherLocale

Commit:
LPS-126375 SF

nkkwon committed:
LPS-126216 Add viewWebContentTitle macro

nkkwon committed:
LPS-126216 Add test for DefaultTitlePersistsUponFirstlyTranslatingForAnotherLocale

tinatian committed:
LPS-127261 Add Log4jUtil#getLogLevelStrings, prepare for next

tinatian committed:
LPS-127261 Apply new method to Log4jUtil#getOriginalLevel

tinatian committed:
LPS-127261 Inline the only usage and deprecate API method

michaelhashimoto committed:
LRCI-1931 Fix logic to find the property values for different suites

tinatian committed:
LPS-127261 Apply Log4jUtil#getLogLevelStrings

chaowang-123 committed:
LPS-126428 Add case for when change translation doesn't change another one

drewbrokke committed:
LPS-125697 commerce-account-service: adds static transform method

drewbrokke committed:
LPS-125697 commerce-account-service: schema: adds reference to AccountEntryOrganizationRel

drewbrokke committed:
LPS-125697 commerce: autogenerated: buildService

Commit:
LPS-126725 Add test CheckDuplicatedRepeatableStructureOnWebContent

drewbrokke committed:
LPS-125697 account-service: service: adds if statement from Commerce service

drewbrokke committed:
LPS-125697 account-service: service: adds delete methods

drewbrokke committed:
LPS-125697 account: autogenerated: buildService

drewbrokke committed:
LPS-125697 account-service: service: adds get methods

Jeremy-Ch committed:
LPS-127173 Update macro for viewing document details

Jeremy-Ch committed:
LPS-127173 Update macro for editing the image field content

Commit:
COMMERCE-5588 dispatch-service - change column type

Commit:
COMMERCE-5588 dispatch-service - update .bnd

Commit:
LPS-127093 Ensure that an user attribute type is returned

Commit:
LPS-127093 Rename subjectNameIdentifierType to userAttributeType

Jeremy-Ch committed:
LPS-127173 Add tests for the DM staging

naraandrade committed:
LPS-127138 Apply editFormView button in both popovers on SelectFormView

naraandrade committed:
LPS-127138 Only select valid form view

victorg1991 committed:
LPS-127297 Do not show grid lines in master page columns

drewbrokke committed:
LPS-125697 account-service: service: adds get by organizationId methods

drewbrokke committed:
LPS-125697 account-service: service: adds method to get by accountEntryId and organizationId

naraandrade committed:
LPS-127138 Prevent invalid prop warning

adolfopa committed:
LPS-127211 Add service method to update storage size

drewbrokke committed:
LPS-125697 account-service: service: adds method to fetch by accountEntryId and organizationId

adolfopa committed:
LPS-127211 Add OSGi command to update storage size manually

drewbrokke committed:
LPS-125697 commerce-account-service: service: bridges service methods

victortrajano committed:
LPS-126323 Add CheckFieldsetTranslationsOnStructure test

drewbrokke committed:
LPS-125697 commerce-account-service: service: bridges additional methods

Commit:
LPS-126368 Add CheckLocalizationOfStructureUponLanguageChange

Commit:
LPS-126368 SF

drewbrokke committed:
LPS-125697 commerce-account-service: adds upgrade process to move CommerceAccountOrganizationRels to AccountEntryOrganizationRel table

drewbrokke committed:
LPS-125697 commerce-account-service: service: removes temporary DSLQuery statements from CommerceAccountLocalService

Commit:
LPS-126371 Add FORM_FIELDS_GROUP_NESTED_LABEL path

Commit:
LPS-126371 Add DuplicateTranslatedField test

Commit:
LPS-126371 SF

juliaduartelira committed:
LPS-126342 Add TranslateWebContentToNLanguages Method

rodrigopaulino committed:
LPS-122906 Upgrade form's settings to remove the versions from the workflow definitions

Commit:
COMMERCE-5588 dispatch-service - fix dispatchTaskSettings too - it is 'TEXT null' not 'LONGTEXT null'

Commit:
LPS-126379 Add TranslationPersistsForTwoLevelNestedFields

Commit:
LPS-126379 SF

Commit:
LPS-126375 Validate default title first

pedro-michel committed:
LPS-126347 Add TranslateWebContentFromStructureWithNLevelNestedFields

RitaSilveira committed:
LPS-126839 Add ShowAndHideRichTextLabel

RitaSilveira committed:
LPS-126839 SF

brittneyq committed:
LPS-126222 Add test for DefaultTitleIsDisplayedUponLocaleChange

rodrigopaulino committed:
LPS-122906 baseline

Commit:
LRQA-63840 Add locators of Publish and Schedule disabled in nav bar

Commit:
LRQA-63840 Add a new condition to goto review publications

Commit:
LRQA-63840 Add a new condition to goto publish publications

Commit:
LRQA-63840 Add a locator for disabled revert button

Commit:
LRQA-63840 Add a test to view out of date publications after upgrading

Commit:
LRQA-63840 Ignore DB2 and oracle as database types

michaelhashimoto committed:
LRCI-1957 Always set build parameters as environment variables for RCA Tool

michaelhashimoto committed:
LRCI-1957 Fix RCA Tool build objects to find the repo & branch name

michaelhashimoto committed:
LRCI-1957 Do not pass down specific types of BatchTestClassGroups where not needed

michaelhashimoto committed:
LRCI-1957 Always enable segment test class groups

michaelhashimoto committed:
LRCI-1957 Add test class groups for RCA tests

Commit:
LRQA-63840 Modify publication status locator and related macros

Commit:
LRQA-63840 Add a macro to view out of date publications

Commit:
LRQA-63840 Update Publications test suite

Commit:
LRQA-63840 Modify the locators order

Hanlf committed:
LPS-127180 Add new locators about URL

Hanlf committed:
LPS-127180 Add and update macros about DM

Hanlf committed:
LPS-127180 Add tests about DM staging

Commit:
LPS-126034 Update alloyeditor to v2.11.16

dgarciasarai committed:
LPS-127312 Clear editingExperience state on modal close

Jeremy-Ch committed:
LPS-127159 Update macros for editing and viewing document custom field

Jeremy-Ch committed:
LPS-127159 Update macro for viewing no related asset in AP

jkappler committed:
LPS-125599 Match namings of pageDefinition

jkappler committed:
LPS-125599 buildR

jkappler committed:
LPS-125599 Adapt util class

gianmarcobrunialti committed:
COMMERCE-5596 Isolate order refresh to call the API by itself on update

Commit:
LPS-126216 Remove ignores and moving openWebContetnAdmin to setup

izaera committed:
LPS-127356 Fix incorrect web context path

izaera committed:
LPS-127356 Support npmResolvedPackageName for webpack projects

Even if it is not strictly necessary because npmResolvedPackageName will be equal
to the module's name this will aid a lot with migration and doesn't harm.

izaera committed:
LPS-127356 Bring dual (bundler+webpack) support to SoyRenderer

joseabelenda committed:
LPS-126032 - auto SF

Commit:
LPS-127349 Set the default value as the placeholder when Submit Label is empty for a certain language

Commit:
LPS-127349 Add input field border

izaera committed:
LPS-127356 SF

Jeremy-Ch committed:
LPS-127159 Add tests for the depot staging

victorg1991 committed:
LPS-125510 Rename methods to fit new class names

victorg1991 committed:
LPS-125510 Replace usages

Commit:
LPS-127093 Rename to authType and use CompanyConstants to associate stronger with the existing portal concept?

drewbrokke committed:
LPS-125697 account-service: moves AccountGroupRel upgrade to a major version

drewbrokke committed:
LPS-125697 account-service: schema: fixes field name case

drewbrokke committed:
LPS-125697 account: autogenerated: buildService

juliaduartelira committed:
LPS-126837 Add RepeatRichTextOnUpload Method

juliaduartelira committed:
LPS-126837 SF

fcodias committed:
LRAC-7406 Order matters

fcodias committed:
LRAC-7406 Fix delay when request fails

Commit:
LPS-126829 Add EmptyAFieldGroup

kellylisbino committed:
LPS-126820 Add REPEATABLE_BUTTON method in WCEditWebContent path

kellylisbino committed:
LPS-126820 Add viewDuplicatedField method in WebContent macro

Commit:
LPS-126829 SF

kellylisbino committed:
LPS-126820 Add addRepeatableFieldGroups method in DataEngine macro

kellylisbino committed:
LPS-126820 Add SetFieldGroupAsRepeatableAndExecuteOnUpload

kellylisbino committed:
LPS-126820 SF

jonathanmccann committed:
LPS-127390 Use correct path

Commit:
LRQA-64776 - Fix DroolsUpgrade and CalendarUpgrade components

michaelhashimoto committed:
LRCI-1931 SF

Commit:
LRQA-64782 Remove unnecessary if

Commit:
LRQA-64782 Modify case to assert success message and test space charactrer

Commit:
LRQA-64782 Simplify case by macro

austinchiang committed:
LPS-126848 Move test to the correct stub

michaelhashimoto committed:
LRCI-1876 Always cache the portal release object for fixpack builds

brianchandotcom committed:
LPS-105380 prep next

michaelhashimoto committed:
LRCI-1876 Get the hotfix information from a hotfix build

brianchandotcom committed:
POSHI-146 prep next

michaelhashimoto committed:
LRCI-1876 Make replace env vars a public method to be used

michaelhashimoto committed:
LRCI-1876 Add method to get portal hotfix release objects from build results

joshchong committed:
LPS-126371 Use more appropriate macro

joshchong committed:
LPS-126371 Use existing macro that covers both cases

joshchong committed:
LPS-126371 Remove unused variable

joshchong committed:
LPS-126371 Remove duplicate locator

michaelhashimoto committed:
LRCI-1876 Add more env variables to be replaced for spira artifacts

michaelhashimoto committed:
LRCI-1876 Move spira notifications to be controlled by properties

joshchong committed:
LPS-126368 Remove unnecessary configuration

liuxingye committed:
LPS-127167 Add tests for publish depot image with latest version in blog

liuxingye committed:
LPS-127166 Add tests for AL staging related with open graph

michaelhashimoto committed:
LRCI-1876 Add properties to configure slack notifications

Commit:
LPS-126375 Add if to make it reusable

Hanlf committed:
LPS-127180 Update macro to make it work for remote staging

Hanlf committed:
LPS-127180 Add tests about DM remote staging

Commit:
LPS-126379 Modify case to assert translation persists for two level nested fields

Commit:
LPS-126371 Modify locator to make it robust

Commit:
LPS-126371 Modify case to assert duplicate translate field

Commit:
LPS-126368 Modify case to check localization of structure upon language change

Commit:
LPS-126390 Modify case to assert translation does not change another

Commit:
LPS-126375 Modify case to assert default title persists upon firstly translating for another locale

liuxingye committed:
LPS-127172 Add path for type in second numeric field

liuxingye committed:
LPS-127172 Add test for create document with duplicate fields

Commit:
LPS-126323 Add macro to edit fieldset label

Commit:
LPS-126323 Remove deprecated locators

Commit:
LPS-126323 Add locator for localization dropdown toggle

Commit:
LPS-126323 SF

Commit:
LPS-126323 Modify macro to change locale in modal

liuxingye committed:
LPS-122814 Modify case to assert empty state text

Commit:
LPS-126323 Modify case to check fieldset translations on structure

liuxingye committed:
LPS-122814 Add path to assert the related question

liuxingye committed:
LPS-122814 Add test for view related question

Commit:
LPS-126725 Add locator for settings text any

Commit:
LPS-126725 Modify locator to make it robust

Commit:
LPS-126725 Modify case to check duplicated repeatable structure on web content

JoyceWang08211 committed:
LRQA-64788 Add test to publish web content folder with child folders

Commit:
LPS-121895 Add missing callback for creation button

Commit:
LPS-121895 Migrate v2 usages in password-policies-admin/password-policies-admin-web

The `<clay:management-toolbar-v2 />` tag has been migrated to the
`<clay:management-toolbar />` tag (which uses clay v3).

Jeremy-Ch committed:
LPS-126865 Add test for the image field

Commit:
LPS-126342 Modify case to assert translate web content to n languages

4lejandrito committed:
LPS-127049 Using the right namespace in the backend too

Without this nothing was being saved after c4b5862a5ae44278bb9a48e22e86d2a53ddcae45.

adolfopa committed:
LPS-127211 Autogenerated

Commit:
LPS-126347 Modify case to assert translate web content from structure with n level nested fields

riccardo-alberti committed:
COMMERCE-5446 Order left joins alphabetically

Commit:
LPS-126216 Changes after review for DefaultTitleIsDisplayedUponLocaleChange

Commit:
LPS-127093 No need to use endsWith() because only discrete values are passed. And "uuid" is not a possible value

Commit:
LPS-127093 Auto SF

dgarciasarai committed:
LPS-127124 Use c-inner CSS class in experience list item

Commit:
LPS-126216 Changes after review for DefaultTitlePersistsUponFirstlyTranslatingForAnotherLocale

Commit:
LPS-125810 Fix error in questions when there are multiple CKEditors and one editor is closed after publishing the new comment or answer

Commit:
LPS-126216 SF

ealonso committed:
LPS-125362 It should be structureVersionId

brianchandotcom committed:
LPS-125256 Hugo SF, avoid casting, and inline

brianchandotcom committed:
LPS-125256 sort

brianchandotcom committed:
LPS-125256 usual order

brianchandotcom committed:
LPS-122906 SF

ambrinchaudhary committed:
LPS-125733 SF

brianchandotcom committed:
LPS-127109 SF

joseabelenda committed:
LPS-126032 - code review

liferay-translation committed:
LPS-77699 Update Translations

brianchandotcom committed:
LPS-125362 Hugo SF

Commit:
LPS-126839 Use editShowOrHideLabel instead of creating a new macro

Commit:
LPS-126839 Remove navigation steps from test case

Commit:
LPS-126839 Use the default document type name defined in the setUp when creating a document type

drewbrokke committed:
LPS-121895 password-policies-admin-web: refactor: simplify (edit props transformer)

Commit:
LPS-126839 Open document via DMNavigator.gotoDocumentCP macro

brianchandotcom committed:
LPS-127211 wordsmith

brianchandotcom committed:
LPS-127211 Hugo SF, like this

drewbrokke committed:
LPS-121895 password-policies-admin-web: refactor: simplify (view props transformer)

brianchandotcom committed:
LPS-127211 This does the same thing.. but faster right? I suppose this will fail if you pass in a wrong company ID, but it would make sense to fail then.

brianchandotcom committed:
LPS-125733 In this case, this is faster. Hugo SF, we have quite a few of these.

brianchandotcom committed:
LPS-125733 In this case, this is faster. Hugo SF, we have quite a few of these... like here

brianchandotcom committed:
LPS-127261 var name should go by entry value and not entry key

brianchandotcom committed:
LPS-127211 consistency

brianchandotcom committed:
LRCI-1931 prep next

austinchiang committed:
LPS-127180 Uncomment test steps

Commit:
LPS-126602 Add new method dismissAlert

Commit:
LPS-126602 SF

joseabelenda committed:
LPS-126032 - auto SF

Commit:
LPS-126837 Use fieldName and index when using assertFieldRepeated

joshchong committed:
LPS-126839 Removing acceptance property. What's the point of having this?

joshchong committed:
LPS-126839 Pass in variables explicitly to be more clear and improve readability

joshchong committed:
LPS-126839 Remove unnecessary variable in macro call

joshchong committed:
LPS-126839 Allow for unhiding hidden labels and apply to test

pyoo47 committed:
LRCI-1961 Truncate long lines and flush to prevent Jenkins freeze

joshchong committed:
LPS-126837 Use correct field name

joshchong committed:
LPS-126837 Use more distinguishable field name

joshchong committed:
LPS-126837 Use more appropriate nav macro

joshchong committed:
LPS-126837 Use clearer document name

joshchong committed:
LPS-126837 Use existing nav macro

joshchong committed:
LPS-126837 SF

brianchandotcom committed:
LRCI-1957 prep next

brianchandotcom committed:
LPS-127476 Wordsmith

NemethNorbert commented:
LGTM

moltam89 commented:
https://github.com/moltam89/liferay-portal/pull/574
